host/bcm283x/gpio.go

```diff
@@ -266,6 +266,16 @@ func (p *Pin) Out(l gpio.Level) error {
 		return err
 	}
 	// Change output before changing mode to not create any glitch.
+	p.FastOut(l)
+	p.setFunction(out)
+	return nil
+}
+
+// FastOut sets a pin output level with Absolutely No error checking.
```

Comment:
We might want to specify what kind of errors could happen and why/when to use that version vs `Out`.

Reply:
Updated comment to call out undefined behavior.